content/en/docs/tasks/configure-pod-container/pull-image-private-registry.md

```diff
@@ -238,6 +236,14 @@ Events:
   ...  FailedToRetrieveImagePullSecret  ...  Unable to retrieve some image pull secrets (<regcred>); attempting to pull the image may not succeed.
 ```
 
+## Using images from multiple registries
+
+A pod can have multiple containers, each container image can be from a different registry.
+You can use multiple `imagePullSecrets` with one pod, and each can contain multiple credentials.
+
+The image pull will be attempted using each credential that matches the registry.
+If no credentials match the registry, the image pull will be attempted without authorization.
```

Comment:
Technically runtimes can be configured to auth against registries without Kubernetes being aware of it. I think mentioning it should be sufficient, like:

```suggestion
If no credentials match the registry, the image pull will be attempted without authorization or
custom runtime specific configuration.
```

Reply:
Added.

I was trying to stay away from the other mechanisms in this section to not confuse things, but I guess there are related links for people to look at for more detail.